proto/hm/sensordata.proto

```diff
@@ -130,4 +130,124 @@ message GnssData {
     RxmMeasx rxm_measx = 24;
     UbxSecEcsign sec_ecsign = 25;
     string sec_ecsign_buffer = 26;
+}
+
+/// Low Level UBX Messages
+
+message NavDop {
+    string system_time = 1;
+    uint32 itow_ms = 2;
+    uint32 gdop = 3;
+    uint32 pdop = 4;
+    uint32 tdop = 5;
+    uint32 vdop = 6;
+    uint32 hdop = 7;
+    uint32 ndop = 8;
+    uint32 edop = 9;
+}
+
+message NavSat {
+    message Svs {
+        uint32 gnss_id = 1;
+        uint32 sv_id = 2;
+        uint32 cno_dbhz = 3;
+        int32 elev_deg = 4;
+        int32 azim_deg = 5;
+        int32 pr_res_me1 = 6;
+        uint32 flags = 7;
+    }
+
+    string system_time = 1;
+    uint32 itow_ms = 2;
+    uint32 version = 3;
+    uint32 num_svs = 4;
+    repeated Svs svs = 5;
+}
+
+message NavPvt {
+    string system_time = 1;
+    uint32 itow_ms = 2;
+    uint32 year_y = 3;
+    uint32 month_month = 4;
+    uint32 day_d = 5;
+    uint32 hour_h = 6;
+    uint32 min_min = 7;
+    uint32 sec_s = 8;
+    uint32 valid = 9;
```

Comment:
valid, flags, and flags2 in this message are uint8's in the u-blox docs, but looks like proto only has uint32 or bytes. I assume we're not resource constrained enough to care about that difference?

Reply:
It should be fine, it wouldn't increase the memory much unless we had millions of elements in redis